datasets/bagh/bagh.json

```diff
@@ -201,6 +232,18 @@
           "stadsdeel"
         ],
         "display": "id",
+        "additionalRelations": {
+          "wijk":  {
+            "table": "wijk",
+            "field": "ggw_gebied",
+            "format": "embedded"
+          },
+          "buurt": {
+            "table": "buurt",
+            "field": "ggw_gebied",
+            "format": "summary"
```

Comment:
Ziet er goed uit Bart, is de keuze voor een embedded of summary format gebaseerd op de hoeveelheid URL's die je kunt verwachten?

Reply:
Dat heeft inderdaad te maken met het verwachte aantal URL's